Tests/NumLib/LocalToGlobalIndexMap.cpp

```diff
@@ -107,12 +107,41 @@ TEST_F(NumLibLocalToGlobalIndexMapTest, DISABLED_SubsetByComponent)
         new MeshLib::MeshSubsets{selected_subset.get()}};
 
     auto dof_map_subset = std::unique_ptr<NumLib::LocalToGlobalIndexMap>{
-        dof_map->deriveBoundaryConstrainedMap(0,  // variable id
-                                              1,  // component id
+        dof_map->deriveBoundaryConstrainedMap(1,  // variable id
+                                              0,  // component id
                                               std::move(selected_component),
                                               some_elements)};
 
     // There must be as many rows as nodes in the input times the number of
     // components.
     ASSERT_EQ(selected_nodes.size(), dof_map_subset->dofSizeWithGhosts());
 }
+
+#ifndef USE_PETSC
+TEST_F(NumLibLocalToGlobalIndexMapTest, MultipleVariablesMultipleComponents)
+#else
+TEST_F(NumLibLocalToGlobalIndexMapTest, DISABLED_MultipleVariablesMultipleComponents)
+#endif
+{
+    // test 2 variables (1st variable with 1 component, 2nd variable with 2 components)
```

Comment:
you are right, Dima told me it already worked if 1st var has 1 comp. I added another test with 1st var with two components.